src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,45 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+	@Test
+	@DisplayName("요구사항1 Split Test")
+	void splitTest() {
+		String[] result = "1,2".split(",");
+		assertThat(result).containsExactly("1","2");
+		result = "1".split(",");
+		assertThat(result).contains("1");
+	}
+	
+	@Test
+	@DisplayName("요구사항 2 Substring Test")
+	void subStringTest() {
+		String result = "";
+		String Goal = "(1,2)";
```

Comment:
변수는 소문자로 시작합니다. 

```suggestion
		String result = "";
		String goal = "(1,2)";
```

`result` 같은 경우는 `""` 공백을 할당했다가 다시 다른 값을 할당하는데요. 그럼 공백을 할당할 이유가 있을까요? 
`final` 키워드를 사용해 보는 것은 어떨까요? 어떤 장점이 있을까요?